components/style/values/computed/length.rs

```diff
@@ -194,6 +194,18 @@ impl From<LengthOrPercentageOrNone> for Option<CalcLengthOrPercentage> {
 
 impl ToCss for CalcLengthOrPercentage {
     fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
+        // FIXME(emilio): This should be consistent with specified calc().
+        //
+        // Also, serialisation here is kinda weird with negative percentages,
+        // for example, as of right now:
+        //
+        //  calc(10px - 5%)
+        //
+        // would serialize as:
+        //
+        //  calc(10px + -5%)
+        //
+        // Need to update and run through try.
```

Comment:
(And I'd need to find a good revision, etc...)

I'll push a try run and if this hasn't landed tomorrow morning I'll just push it.